Sources/Common/Core/CellArray/index.d.ts

```diff
@@ -35,12 +35,6 @@ export interface vtkCellArray extends vtkDataArray {
 	 */
 	getCell(loc: any): void;
 
-	/**
-	 * Reset the cell array by setting the number of cells to 0.
-	 * NOTE: This won't touch the actual memory of the underlying typedArray.
-	 */
-	initialize(): void;
```

Comment:
This is a breaking change, so the commit message should reflect this.

Reply:
It is actually not a breaking change, `initialize()` is only removed from CellArray typescript definition but it is still available in superclass typescript definition.